tests/test_modis.py

```diff
@@ -657,8 +657,8 @@ def test_create(self):
             dates = hdf5_file.get("dates")
             self.assertTrue(dates)
 
-        self.assertNotEqual(smtH5.last_collected, "2002201")
-        self.assertEqual(smtH5.last_collected, "2002209")
+        self.assertNotEqual(smtH5.last_smoothed, "2002201")
```

Comment:
so here you basically just renamed it - not sure if that fit's the purpose.

Originally this was meant to check the last collected in the file, now it'll check the date within the smooth file. From the top of my head I'm not certain if that is what we want, or if the test would actually pass.

Ideally we'd test both properties, and even the consistency between the properties

Reply:
As you can see, the original code checked the last_collected property: behind the property, code was retrieving the last date from the smoothed H5 `rawdates` attribute. This property was thus renamed to "last_smoothed". Hence, reference to this renamed property is what you find here. The _new_ property "last_collected" digs into the raw H5 archive -- to me, this seemed not to be the scope of this test... Proposing to leave it like this -- closest to the original code.